wordpress/wp-content/plugins/cds-base/classes/Modules/Forms/Contact/ContactForm.php

```diff
@@ -162,27 +86,27 @@ class="gc-input-text"
                     </div>
 
                     <div class="focus-group">
-                    <?php $this->checkBox(
+                    <?php echo Utils::checkboxField(
```

Comment:
1) Wondering if we can remove all these opening and closing PHP tags now that this is cleaned up.
2) What do you thinking about echo 'ing but default and having an optional param to return vs echo (might help the readability)